### PR TITLE
Feature/iplb status from api

### PR DESCRIPTION
**Pull request by Mboulianne**

### CODE REVIEW

File: client/app/dbaas/dbaas-metrics/dashboard/metrics-dashboard.controller.js [hunk 1]
```diff
@@ -98,6 +98,11 @@
                     text: this.$translate.instant("common_manage"),
                     href: this.ControllerHelper.navigation.getUrl("contacts", { serviceName: this.serviceName }),
                     isAvailable: () => true
+                },
+                editName: {
+                    text: this.$translate.instant("metrics_tiles_modify"),
```
Comment (whispyy) [inline client/app/dbaas/dbaas-metrics/dashboard/metrics-dashboard.controller.js]: One day we should update translations with common ones, I think metrics_tiles_modify equals to common_edit

File: client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js [hunk 2]
```diff
@@ -3,24 +3,68 @@ angular.module("managerApp")
...
             description: "<",
             actions: "<",
             type: "@"
         }
+    }))
+    .directive("cuiTileTermStatus", () => ({
+        replace: true,
+        restrict: "E",
+        controllerAs: "$ctrl",
+        controller: class CuiTileDefinitionTermCtrl {},
```
Comment (whispyy) [inline client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js]: you can remove class name just : class {},
this create an anonyme class

File: client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js [hunk 3]
```diff
@@ -3,24 +3,68 @@ angular.module("managerApp")
...
+            <dt class="cui-tile-item-status__term" data-ng-bind="$ctrl.term"></dt>`,
+        bindToController: {
+            term: "<"
+        }
+    }))
+    .directive("cuiTileDescriptionStatus", () => ({
+        replace: true,
+        restrict: "E",
+        controllerAs: "$ctrl",
+        controller: class CuiTileDefinitionDescriptionCtrl {},
```
Comment (whispyy) [inline client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js]: same

File: client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js (hunk 2 above)
Comment (Mboulianne) [reply client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js]: Done

File: client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js (hunk 3 above)
Comment (Mboulianne) [reply client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js]: Done

**Pull request merged**